test/onnx/test_operators.py

```diff
@@ -915,6 +916,12 @@ def test_dynamic_axes_add(self):
                         dynamic_axes={"input_1": {1: "dim_1"}, "input_2": {1: "dim_2"}},
                         opset_version=12)
 
+    def test_dynamic_axes_add_2(self):
```

Comment:
I'd name it `test_dynamic_axes_add_inputs_same_symbolic_shape` and remove the comment. It's OK to have a long name for the test since it's not used in any other place and it's nice to understand what it's meant to test.

Reply:
done.